app/src/main/java/it/ministerodellasalute/immuni/ui/dialog/BottomSheetDialogLightFragment.kt

```diff
@@ -52,6 +53,7 @@ abstract class BottomSheetDialogLightFragment : BottomSheetDialogFragment() {
         if (window != null) {
             val metrics = DisplayMetrics()
             window.windowManager.defaultDisplay.getMetrics(metrics)
+            window.addFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)
```

Comment:
@eliseomartelli could you please provide some screenshot to show me the difference please?

Reply:
Yes, I've added the screenshot inside the pr message.

Reply:
@eliseomartelli this cause the bottom sheet to go behind the status bar too, and this is not desired. Tested on my Pixel 3a with Android 10.

Reply:
![device-2020-05-26-163836](https://user-images.githubusercontent.com/3175499/82914121-68ffc400-9f6f-11ea-8bdf-5b8f7dda6462.png)

Reply:
Oh, ok... 
Maybe we can tweak this line to account for half of the height of the navigation bar? 

https://github.com/immuni-app/immuni-app-android/blob/cc848f7b0dec4f6681f214117667491bfffbe8d7/app/src/main/java/it/ministerodellasalute/immuni/ui/dialog/BottomSheetDialogLightFragment.kt#L63